server/protocols/webadmin-cli/pom.xml

```diff
@@ -45,6 +45,16 @@
             <artifactId>picocli</artifactId>
             <version>4.5.1</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.openfeign</groupId>
+            <artifactId>feign-core</artifactId>
+            <version>11.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.openfeign</groupId>
+            <artifactId>feign-gson</artifactId>
+            <version>10.11</version>
+        </dependency>
```

Comment:
Can we rather use https://mvnrepository.com/artifact/io.github.openfeign/feign-jackson/11.0 ?

(we use jackson in other parts of James)